### PR TITLE
Link to new extension type specification

### PR DESCRIPTION
**Pull request by parlough**
Avoiding confusion and out-of-date information for visitors to the previous iterations.

\cc @eernstg

### CODE REVIEW

File: working/1426-extension-types/feature-specification.md [hunk 1]
```diff
@@ -2,7 +2,13 @@
 
 Author: eernst@google.com
 
-Status: Draft
+Status: Obsolete
+
+**This proposal is now obsolete. The mechanism has been renamed to
+_extension types_. Please see the new [extension types specification][] for the
+most recent proposed specification.**
```
Comment (eernstg) [inline working/1426-extension-types/feature-specification.md:9]: Perhaps:
```suggestion
**This proposal is now obsolete. The mechanism has been renamed multiple times, and the
current name is again _extension types_. Please see the new 
[extension types specification][] for the accepted specification proposal.**
```

File: working/1426-extension-types/feature-specification-inline-classes.md [hunk 2]
```diff
@@ -2,8 +2,13 @@
 
 Author: eernst@google.com
 
-Status: Accepted.
+Status: Obsolete
 
+**This proposal is now obsolete. The mechanism has been renamed to
+_extension types_. Please see the [extension types specification][] for the
+most recent proposed specification.**
```
Comment (eernstg) [inline working/1426-extension-types/feature-specification-inline-classes.md:9]: Perhaps:
```suggestion
**This proposal is now obsolete. The mechanism has been renamed to
_extension types_. Please see the [extension types specification][] for the
accepted specification proposal.**
```

**Comment by parlough** [comment]
Thanks for the suggestions, those make sense to me, applied :)

**Comment by parlough** [comment]
Would you mind merging these updates? I don't have commit access. Thanks so much!

**Pull request merged**